gui/pagination.go

```diff
@@ -62,7 +67,13 @@ func (ui *GUI) paginatedBlocks(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	count, confirmedWork := ui.cache.getConfirmedMinedWork(first, last)
+	count, confirmedWork, err := ui.cache.getConfirmedMinedWork(first, last)
+
```

Comment:
Any reason why you're spacing the err check blocks from the call (seen multiple instances of this here) ? Seems unnecessary.